File: Web/RMA-Web-Shell/projects/medicare/src/app/medical-invoice-manager/models/invoice-line-under-assess-reason.ts
import { BaseClass } from 'projects/shared-models-lib/src/lib/common/base-class';

export class InvoiceLineUnderAssessReason extends BaseClass {
    invoiceLineId: number| null;
    tebaInvoiceLineId: number| null;
    underAssessReasonId: number;
    underAssessReason: string;
    comments: string;
}

File: Service/MediCare/RMA.Service.MediCare.Contracts/Entities/Medical/InvoiceLineUnderAssessReason.cs
using RMA.Common.Entities;

namespace RMA.Service.MediCare.Contracts.Entities.Medical
{
    public class InvoiceLineUnderAssessReason : AuditDetails
    {
        public int? InvoiceLineId { get; set; }
        public int? TebaInvoiceLineId { get; set; }
        public int UnderAssessReasonId { get; set; }
        public string UnderAssessReason { get; set; }
        public string Comments { get; set; }
    }
}
